client/styles.scss

```diff
@@ -0,0 +1,2 @@
+// Import full Spectre source code
+@import "node_modules/spectre.css/src/spectre";
```

Comment:
for this app i used a CSS framework called Spectre CSS, there is not much of my own styling in this app, but in prod apps i generally like to customize the styling.